src/application.ts

```diff
@@ -10,18 +10,23 @@ import {
 import {
   Simple,
   templateFactory,
-  RenderResult
+  RenderResult,
+  ComponentDefinition,
+  Component
 } from '@glimmer/runtime';
 import {
   UpdatableReference
 } from '@glimmer/object-reference';
+import {
+  Option
+} from '@glimmer/util';
 import ApplicationRegistry from './application-registry';
 import DynamicScope from './dynamic-scope';
 import Environment from './environment';
+import mainTemplate from './templates/main';
```

Comment:
@danielrosenwasser suggested we just put stub `.d.ts` files in places where the build tools will generate a file.